docs/applications/containers/deploy-nodebalancers-with-linode-ccm/index.md

```diff
@@ -25,7 +25,7 @@ Using the Linode Cloud Controller Manager to create NodeBalancers will create bi
 
 ## Before You Begin
 
-You should have a working knowledge of Kubernetes and familiarity with the `kubcetl` command line tool before attempting the instructions found in this guide. For more information about Kubernetes, consult our [Kubernetes Beginner's Guide](/docs/applications/containers/beginners-guide-to-kubernetes/) and our [Getting Started with Kubernetes](/docs/applications/containers/getting-started-with-kubernetes/) guide.
+You should have a working knowledge of Kubernetes and familiarity with the `kubcetl` command line tool before attempting the instructions found in this guide. For more information about Kubernetes, consult our [Kubernetes Beginner's Guide](/docs/applications/containers/beginners-guide-to-kubernetes/) and our [Getting Started with Kubernetes](/docs/applications/containers/kubernetes/getting-started-with-kubernetes/) guide.
```

Comment:
All of the kubernetes guides have moved to /docs/kubernetes, they no longer exist at /docs/applications/containers/kubernetes. Can you change the links to point to their new locations? This would need to be done in each of the guides.